src/Discovery/DiscoveryActionBar/components/OpenFillRequestFormButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { FileTextOutlined } from '@ant-design/icons';
+import { Popover, Button } from 'antd';
+
+const OpenFillRequestFormButton = ({
+    props,
+  }) => (
+  props.config.features.exportToWorkspace?.openFillRequestForm && (
+    <Popover
+      className='discovery-popover'
+      arrowPointAtCenter
+      title={(
+        <React.Fragment>
+          &nbsp;
+          <a target='_blank' rel='noreferrer' href='https://pandemicresponsecommons.org'>
+            {'Pandemic Response Commons Website'}
+          </a>.
+        </React.Fragment>
+      )}
+      content={(
+        <span className='discovery-popover__text'>
+          After filling the request form, after your search selection, you would get an approval after which you can use the Gen3 Client
+          to download the data from the selected studies to your local computer.
+        </span>
+      )}
+    >
+      <Button
+        onClick={props.config.features.exportToWorkspace?.fillRequestFormURL ? () => {
+          let combinedIds = '';
+          props.discovery.selectedResources.forEach((item) => {
+            if (combinedIds !== '') {
+              combinedIds += ',';
+            }
+            combinedIds += item._medical_sample_id;
+          });
+          const url = `${props.config.features.exportToWorkspace.fillRequestFormURL}?query=${encodeURIComponent(combinedIds)}`;
+          window.open(url, '_blank');
+        } : () => {
+          window.open(props.config.features.exportToWorkspace.fillRequestFormURL, '_blank');
```

Comment:
why it should open this URL in this case, where `fillRequestFormURL` doesn't exists?

Reply:
Technically it should not redirect anywhere else as there is nothing setup

Reply:
if `fillRequestFormURL` doesn't exist, hide the button request Access

Reply:
I have added the code to hide button `fillRequestFormURL` [here](https://github.com/urvi-occ/data-portal/blob/56b658928439ed424a77b866523c3a49854e688b/src/Discovery/DiscoveryActionBar/components/OpenFillRequestFormButton.tsx#L10)

Reply:
Even though you have put checks at the beginning of this function, I would still strongly suggest you should change this line of code. The falsy expression of this ternary operator should simply just be `() => {}`, the current way the code was written is very confusing

Reply:
This is not a ternary operator it is a URL with parameters. I am really confused here on what this comment means

Reply:
I have changed the code, which is now on L45 let me know if this make sense

Reply:
honestly the simplest way to fix this is to remove L46 entriely, which mean if `fillRequestFormURL` doesn't exist, then this button does nothing

Reply:
Fonctionality-wise this doen't change anything, since this button won't be rendered in the first place if `fillRequestFormURL` doesn't exist. So you could even consider remove the ternary operator from L41 entirely, which will make the code simpler and easier to understand

Reply:
Can you please provide the exact code that you think will be a good here? I can review and add it in PR.